Week1/homework/js-exercises/roundNumber.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+var z = 7.25;
+console.log(z);
+var a = Math.floor(z);
+console.log(a);
+
+if(a > z) {
+    console.log(a);
+}
+
+else {console.log(z);}
```

Comment:
`const` here instead of `var` please. And also, minor indentation problem with the `if-else` block